homeassistant/components/tuya/manifest.json

```diff
@@ -43,5 +43,5 @@
   "integration_type": "hub",
   "iot_class": "cloud_push",
   "loggers": ["tuya_iot"],
-  "requirements": ["tuya-device-sharing-sdk==0.1.9"]
+  "requirements": ["tuya-device-sharing-sdk==0.2.1"]
```

Comment:
Dependency bumps need to go into their own separate PR. Please split this out from this pull request.

../Frenck

Reply:
OK. Split this off into #130333. I'll update this PR once that change has been merged